pages/references.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div id="main-content" class="text-wrapper">
+    <h1 class="text-center mb-6">Project References</h1>
+    <p class="mb-3">
+      <span class="font-medium">Database Source:</span> Julian Goodare, Lauren
+      Martin, Joyce Miller and Louise Yeoman,
+      <a
+        class="underline hover:text-gray-500"
+        href="http://www.shca.ed.ac.uk/witches/"
+        target="_blank"
+      >
+        'The Survey of Scottish Witchcraft' </a
+      >(archived January 2003)
+    </p>
+    <br />
```

Comment:
I don't think breaks need a closing slash

Reply:
prettier adds this in

Reply:
huh. Wonder why it does that. Ok, happy to leave as not worth messing around with Prettier configs. Thanks Ruby